patches/@tamagui+progress+1.94.4.patch

```diff
@@ -0,0 +1,11 @@
+diff --git a/node_modules/@tamagui/progress/types/Progress.d.ts b/node_modules/@tamagui/progress/types/Progress.d.ts
+index 58f3db4..cbdfac4 100644
+--- a/node_modules/@tamagui/progress/types/Progress.d.ts
++++ b/node_modules/@tamagui/progress/types/Progress.d.ts
+@@ -254,5 +254,5 @@ declare const Progress: React.ForwardRefExoticComponent<Omit<import("@tamagui/co
+     }, import("@tamagui/core").StaticConfigPublic>;
+ };
+ export { createProgressScope, Progress, ProgressIndicator };
+-export type { ProgressProps, ProgressIndicatorProps };
++export type { ProgressProps, ProgressExtraProps, ProgressIndicatorProps };
```

Comment:
Temporarily using `patch-package` but hoping to send PR to Tamagui. W/o "ExtraProps" exported, it is not possible to re-export Tamagui-derived type with "declaration: true".

Issue that is encountered: https://github.com/microsoft/TypeScript/issues/5711